### PR TITLE
pkg/master: Fix static check failures

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:

/kind cleanup

**What this PR does / why we need it**:

Fix static check failures

**Which issue(s) this PR fixes**:

Ref: #81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/master/master_test.go [hunk 1]
```diff
@@ -254,14 +254,14 @@ func TestGetNodeAddresses(t *testing.T) {
 	addressProvider := nodeAddressProvider{fakeNodeClient}
 
 	// Fail case (no addresses associated with nodes)
-	nodes, _ := fakeNodeClient.List(metav1.ListOptions{})
+	fakeNodeClient.List(metav1.ListOptions{})
```
Comment (neolit123) [inline pkg/master/master_test.go]: this call might not be needed at all, given the results are not used?

**Comment by hwdef** [comment]
@neolit123 
Thanks for your review.

**Comment by BenTheElder** [comment]
/approve
For hack/

**Pull request merged**